src/utils/logger.ts

```diff
@@ -1,58 +1,98 @@
-import { Logger } from 'tslog'
```

Comment:
Since we are replacing tslog, does it need to be removed from the `package.json` or is it a builtin library?